notesapi/v1/tests/test_views.py

```diff
@@ -65,6 +66,16 @@ def _create_annotation(self, **kwargs):
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
         return response.data.copy()
 
+    def _create_annotation_without_assert(self, **kwargs):
+        """
+        Create annotation but do not check for 201
+        """
+        opts = self.payload.copy()
+        opts.update(kwargs)
+        url = reverse('api:v1:annotations')
+        response = self.client.post(url, opts, format='json')
+        return response
```

Comment:
```python
def _create_annotation(self, check_status=True, **kwargs):
        """
        Create annotation
        """
        opts = self.payload.copy()
        opts.update(kwargs)
        url = reverse('api:v1:annotations')
        response = self.client.post(url, opts, format='json')
        if check_status:
            self.assertEqual(response.status_code, status.HTTP_201_CREATED)
        return response.data.copy()
```

@ehteshamkafeel as per my understanding this should work. you don't need to change all the `_create_annotation` calls